tests/test_modeling_tf_common.py

```diff
@@ -276,12 +276,19 @@ def test_onnx_runtime_optimize(self):
     def test_keras_save_load(self):
         config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
 
+        # `CLIPModelTester/TFCLIPModelTester` set `return_loss` to `True`, however this causes the preparation
+        # of `symbolic_inputs` below failed.
+        if self.__class__.__name__ == "TFCLIPModelTest":
+            inputs_dict.pop("return_loss", None)
+
```

Comment:
This should not be in the common tests, but the clip tester should overwrite this test.

Reply:
OK